src/style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.flexbox {
+  display: flex;
+  align-items: flex-end;
+}
+
+.between {
+  justify-content: space-between;
+}
+
+.margins {
+  margin-left: 15px;
+  margin-right: 15px;
```

Comment:
Better to write it in one line 
`margin: 0 15px;`

Reply:
this variant will not work due to conflict with margin-top and margin-bottom in other`s rules